static/gsAdmin/components/customers/customerOverview.spec.tsx

```diff
@@ -367,7 +367,7 @@ describe('CustomerOverview', function () {
     expect(screen.queryByText('Seer:')).not.toBeInTheDocument();
   });
 
-  it('renders no product trials for non-self-serve account', function () {
+  it('renders product trials for non-self-serve account', function () {
     const organization = OrganizationFixture();
     const enterprise_subscription = SubscriptionFixture({
```

Comment:
i realized we can just use `InvoicedSubscriptionFixture` instead of having to overwrite `canSelfServe` here